contracts/extensions/VotingReputation.sol

```diff
@@ -1075,22 +1079,22 @@ contract VotingReputation is ColonyExtension, PatriciaTreeProofs, BasicMetaTrans
     //   of 0x20 represents advancing one byte, 4 is the function signature.
     // So: 0x[length][sig][args...]
 
-    bytes32 functionSignature;
+    bytes4 functionSignature = CHANGE_FUNCTION_SIG_1;
+
     uint256 permissionDomainId;
     uint256 childSkillIndex;
     uint256 expenditureId;
     uint256 storageSlot;
 
     assembly {
-      functionSignature := mload(add(action, 0x20))
       permissionDomainId := mload(add(action, 0x24))
       childSkillIndex := mload(add(action, 0x44))
       expenditureId := mload(add(action, 0x64))
       storageSlot := mload(add(action, 0x84))
     }
 
     // If we are editing the main expenditure struct
-    if (storageSlot == 25) {
+    if (storageSlot == 25 || getSig(action) == CHANGE_FUNCTION_SIG_2) {
```

Comment:
Hmm. It's a good point, but parsing the action for the slots and executing per-slot claim delay changes seems extreme. It would also not work with the current implementation and introduces a potential gas issue -- if someone is attempting to change 100 slots, do we fire off 100 external calls to set claim delays? That seems nuts.

Reply:
Is the solution another function that has the ability to alter multiple payouts, but only for a single slot?

Reply:
I suppose we could do that, but can we put it in another PR?

Reply:
These functions are being changed in this PR, so it makes sense to do it here, no?